### PR TITLE
Adding BillPayment API Support

### PR DESCRIPTION
**Pull request by Sravanksk**

### CODE REVIEW

File: qbosdk/apis/company_info.py [hunk 1]
```diff
@@ -19,4 +19,4 @@ def get(self):
         Returns:
             Dict in Preferences schema.
         """
-        return self._get_request('CompanyInfo', CompanyInfo.GET_COMPANY_INFO.format(self.__realm_id))
+        return self._get_request(CompanyInfo.GET_COMPANY_INFO.format(self.__realm_id))
```
Comment (Shwetabhk) [inline qbosdk/apis/company_info.py:22]: Why is this change needed? Won't this break stuff?

File: qbosdk/apis/preferences.py [hunk 2]
```diff
@@ -15,4 +15,4 @@ def get(self):
         Returns:
             Dict in Preferences schema.
         """
-        return self._get_request('Preferences', Preferences.GET_PREFERENCES)
+        return self._get_request(Preferences.GET_PREFERENCES)
```
Comment (Shwetabhk) [inline qbosdk/apis/preferences.py:18]: Why is this change needed? Won't this break stuff?

File: setup.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name='qbosdk',
-    version='0.6.1',
+    version='0.7',
```
Comment (Shwetabhk) [inline setup.py:11]: 0.7.0

File: qbosdk/apis/bill_payments.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    def post(self, data: Dict):
+        """
+        Post Bill Payment to Quickbooks Online
+        :param data: Dict in Bill Payment schema
+        :return:
+        """
+        return self._post_request(data, BillPayments.POST_BILL_PAYMENT)
+
+    def delete(self, bill_payment_id: str):
```
Comment (Shwetabhk) [inline qbosdk/apis/bill_payments.py:30]: Have you tested delete?

File: qbosdk/apis/api_base.py [hunk 5]
```diff
@@ -85,7 +85,7 @@ def _query_get_all(self, object_type: str, url: str) -> List[Dict]:
 
         raise QuickbooksOnlineSDKError('Error: {0}'.format(response.status_code), response.text)
 
-    def _get_request(self, object_type: str, api_url: str) -> List[Dict] or Dict:
```
Comment (Shwetabhk) [inline qbosdk/apis/api_base.py:88]: Why remove object_type, are you sure it won't break anything in the apps?

File: qbosdk/apis/preferences.py (hunk 2 above)
Comment (Shwetabhk) [reply qbosdk/apis/preferences.py:18]: https://github.com/fylein/fyle-qbo-api/blob/c4ee4dc2e252d92edbc6e9ad061c33f09c662aa6/apps/quickbooks_online/utils.py#L458

This will break no, we're not using ['Preferences'] key here

File: qbosdk/apis/bill_payments.py (hunk 4 above)
Comment (Sravanksk) [reply qbosdk/apis/bill_payments.py:30]: Yes - tested and working fine

File: qbosdk/apis/api_base.py (hunk 5 above)
Comment (Sravanksk) [reply qbosdk/apis/api_base.py:88]: Yes, this change won't break anything in the apps
object_type is not needed, the API response is same even after removing that param

**Review by Shwetabhk — APPROVED** [review]
